.nuspec/Microsoft.Maui.Controls.MultiTargeting.targets

```diff
@@ -32,10 +32,6 @@
   <ItemGroup Condition="$(TargetFramework.Contains('-windows')) != true ">
     <Compile Remove="**\*.Windows.cs" />
     <None Include="**\*.Windows.cs" />
-    <Compile Remove="**\*.xaml.cs" />
-    <None Include="**\*.xaml.cs" />
-    <Compile Remove="**\*.xaml" />
-    <None Include="**\*.xaml" />
```

Comment:
Since the merge of WinUI, the condition around this block makes this run always, and thus XAML never reaches the actual app. We probably want to do this, but in another way. Removing *.xaml means no XAML at all.